docs/how-to-install.md

```diff
@@ -6,145 +6,170 @@ license: "This software is licensed under the Apache License version 2."
+      |── libs/                                   # Gravitino server dependencies libraries.
+      |── logs/                                   # Gravitino server and Gravitino Iceberg REST server logs. Automatically created after the server starts.
+      |── data/                                   # Default directory for the Gravitino server to store data.
+      |── iceberg-rest-server/                    # Gravitino Iceberg REST server package and dependencies libraries.
+      └── scripts/                                # Extra scripts for Gravitino.
+   ```
 
-The Gravitino server configuration file is `conf/gravitino.conf`. You can configure the Gravitino server by modifying this file. Basic configurations have already been added to this file. All the configurations are listed in [Gravitino Server Configurations](./gravitino-server-config.md).
+### Initialize the RDBMS (Optional)
 
-#### Configure the Apache Gravitino server log
+If you want to use the `relational` backend storage, you need to initialize the RDBMS.
+For details on initializing the RDBMS, see [How to use relational backend storage](./how-to-use-relational-backend-storage.md).
 
-The Gravitino server log configuration file is `conf/log4j2.properties`. Gravitino uses Log4j2 as the Logging system. You can [Log4j2](https://logging.apache.org/log4j/2.x/) to do the log configuration.
+### Configure the Gravitino server
 
-#### Configure the Apache Gravitino server environment
+The `conf/gravitino.conf` file provides basic configurations for the Gravitino server.
+To configure the Gravitino server, you can update this file.
+For a full list of the Gravitino server configurations, see [Gravitino Server Configurations](./gravitino-server-config.md).
 
-The Gravitino server environment configuration file is `conf/gravitino-env.sh`. Gravitino exposes several environment variables. You can modify them in this file.
+### Configure the Gravitino server log
 
-#### Configure Apache Gravitino catalogs
+Gravitino uses [Log4j2](https://logging.apache.org/log4j/2.x/) as the logging system.
+The `conf/log4j2.properties` file provides the Gravitino server log configurations.
+To configure the Gravitino server log, you can update this file.
 
-Gravitino supports multiple catalogs. You can configure the catalog-level configurations by modifying the related configuration file in the `catalogs/<catalog-provider>/conf` directory. The configurations you set here apply to all the catalogs of the same type you create.
+### Configure the Gravitino server environment variables
 
-For example, if you want to configure the Hive catalog, you can modify the file `catalogs/hive/conf/hive.conf`. The detailed configurations are listed in the specific catalog documentation.
+Gravitino exposes several environment variables through the `conf/gravitino-env.sh` file.
+To configure these environment variables, you can update this file.
```

Comment:
Perhaps something like:
Gravitino provides a set of environment variables defined in the `conf/gravitino-env.sh` file. You can configure these environment variables by editing this file.